server/src/test/java/org/elasticsearch/action/synonyms/PutSynonymsActionResponseSerializingTests.java

```diff
@@ -13,8 +13,8 @@
 
 import java.io.IOException;
 
-import static org.elasticsearch.action.synonyms.PutSynonymsAction.Response.Result.CREATED;
-import static org.elasticsearch.action.synonyms.PutSynonymsAction.Response.Result.UPDATED;
+import static org.elasticsearch.synonyms.SynonymsManagementAPIService.UpdateSynonymsResult.CREATED;
```

Comment:
Nitpick: Would `Upsert` make more sense than `Update` since it can have the status of `CREATED`?